Framework/Core/Windows/Devices/Pwm/PwmController.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Windows.Devices.Pwm.Provider;
+
+namespace Windows.Devices.Pwm
+{
+    public sealed class PwmController
+    {
+        private IPwmControllerProvider m_provider;
+        private double m_minFrequency;
+        private double m_maxFrequency;
+
+        internal PwmController(IPwmControllerProvider provider)
+        {
+            m_provider = provider;
+            m_minFrequency = provider.MinFrequency;

```

Comment:
Cached these since we compare with them later.  They should remain static.  We may want to also cache other values that are expected to never change, but I don't want to bloat the object too much.